Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ViewExtensions/ViewExtensionsCode.bind

```diff
@@ -9,7 +9,7 @@
       extensions:StatusBar.BackgroundColor="Blue"
       extensions:StatusBar.BackgroundOpacity="0.8"
       extensions:StatusBar.ForegroundColor="White"
-      extensions:StatusBar.IsVisible="@[IsVisible:Bool:false]"
+      extensions:StatusBar.IsVisible="@[StatusBar IsVisible:Bool:false]"
```

Comment:
Yes it does.. I just wanted it to be clearer on Sample page that its for status bar